plugin/src/main/java/com/google/errorprone/xplat/checker/XplatBans.java

```diff
@@ -174,7 +181,8 @@ private void getJsonData(String json, String fileName) {
    */
   public XplatBans(ErrorProneFlags flags) {
     try {
-      getJsonData(Resources.toString(Resources.getResource("Xplatbans.json"), Charsets.UTF_8),
+      getJsonData(
+          Resources.toString(Resources.getResource("Xplatbans.json"), StandardCharsets.UTF_8),
```

Comment:
It seems like readResourceAsString() does the same thing that this line does. Should this be changed to use readResourceAsString(), or vice versa, for consistency?

Reply:
Good call. Changed to use the helper now. Note that `Resources.getResource()` is a bit different from `getClass().getResource()`: the former actually allows you to read anywhere from the resource where the later confines you to the current class' package (so say `com/google/errorprone/xplat/checker/`), so I also updated the helper to use the former.